ledger-core/v2/network/pulsenetwork/distributor.go

```diff
@@ -52,6 +52,12 @@ type distributor struct {
 	responseHandler future.PacketHandler
 }
 
+type panicHandler struct{}
```

Comment:
`panicHandler` - it's something that handles panics. Use more descriptive name: `panicingHandler`, `handlerThatPanics` or name of your choice that suites.